source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Configuration/HealthCheckConfiguration.cs

```diff
@@ -169,11 +169,11 @@ private static void ConfigureDomainEventsCharges(IServiceCollection serviceColle
                     _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargesDomainEventTopicName),
                     _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargeInformationOperationsAcceptedPersistHistorySubscriptionName),
                     name: "ChargeInformationOperationsAcceptedPersistHistorySubscriptionExists")
-                .AddAzureServiceBusSubscription(
-                    _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.DataHubListenerConnectionString),
-                    _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargesDomainEventTopicName),
-                    _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargePriceCommandReceivedSubscriptionName),
-                    name: "ChargePriceCommandReceivedSubscriptionExists")
+                // .AddAzureServiceBusSubscription(
+                //     _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.DataHubListenerConnectionString),
+                //     _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargesDomainEventTopicName),
+                //     _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargePriceCommandReceivedSubscriptionName),
+                //     name: "ChargePriceCommandReceivedSubscriptionExists")
```

Comment:
I suggest we fix the problem instead 😎
```suggestion
                .AddAzureServiceBusSubscription(
                    _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.DataHubManagerConnectionString),
                    _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargesDomainEventTopicName),
                    _ => EnvironmentHelper.GetEnv(EnvironmentSettingNames.ChargePriceCommandReceivedSubscriptionName),
                    name: "ChargePriceCommandReceivedSubscriptionExists")
```